.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
 - 8
+- 10
+- 12
```

Comment:
Just 12?

Reply:
I think we should do 10 and 12, but drop 8